### ISSUE TITLE
The 1.2 release crashes Sonar 5.1.latest

### ISSUE THREAD

**Issue by zenderol** [arthepsy/sonar-scala-extra#3]
After installing the 1.2 release on _SonarQube 5.1.latest_, the SonarQube application quits with the following exception. 

```
2015.10.12 20:45:45 ERROR web[o.a.c.c.C.[.[.[/]] Exception sending context initialized event to listener instance of class org.sonar.server.platform.PlatformServletContextListener
java.lang.IllegalArgumentException: Null InputStream is not a valid argument
    at com.ctc.wstx.stax.WstxInputFactory.createSR(WstxInputFactory.java:614) ~[woodstox-core-lgpl-4.4.0.jar:4.4.0]
    at com.ctc.wstx.stax.WstxInputFactory.createXMLStreamReader(WstxInputFactory.java:324) ~[woodstox-core-lgpl-4.4.0.jar:4.4.0]
    at org.codehaus.staxmate.SMInputFactory.createStax2Reader(SMInputFactory.java:216) ~[staxmate-2.0.1.jar:na]
    at org.codehaus.staxmate.SMInputFactory.rootElementCursor(SMInputFactory.java:314) ~[staxmate-2.0.1.jar:na]
    at eu.arthepsy.sonar.plugins.scapegoat.rule.GenericRulesParser.parse(GenericRulesParser.java:55) ~[na:na]
    at eu.arthepsy.sonar.plugins.scapegoat.rule.ScapegoatQualityProfile.createProfile(ScapegoatQualityProfile.java:79) ~[na:na]
    at org.sonar.server.qualityprofile.RegisterQualityProfiles.profilesByLanguage(RegisterQualityProfiles.java:182) ~[sonar-server-5.1.jar:na]
    at org.sonar.server.qualityprofile.RegisterQualityProfiles.start(RegisterQualityProfiles.java:91) ~[sonar-server-5.1.jar:na]
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_45-internal]
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_45-internal]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_45-internal]
    at java.lang.reflect.Method.invoke(Method.java:497) ~[na:1.8.0_45-internal]
    at org.picocontainer.lifecycle.ReflectionLifecycleStrategy.invokeMethod(ReflectionLifecycleStrategy.java:110) ~[picocontainer-2.14.3.jar:na]
    at org.picocontainer.lifecycle.ReflectionLifecycleStrategy.start(ReflectionLifecycleStrategy.java:89) ~[picocontainer-2.14.3.jar:na]
    at org.picocontainer.injectors.AbstractInjectionFactory$LifecycleAdapter.start(AbstractInjectionFactory.java:84) ~[picocontainer-2.14.3.jar:na]
    at org.picocontainer.behaviors.AbstractBehavior.start(AbstractBehavior.java:169) ~[picocontainer-2.14.3.jar:na]
    at org.picocontainer.behaviors.Stored$RealComponentLifecycle.start(Stored.java:132) ~[picocontainer-2.14.3.jar:na]
    at org.picocontainer.behaviors.Stored.start(Stored.java:110) ~[picocontainer-2.14.3.jar:na]
    at org.picocontainer.DefaultPicoContainer.potentiallyStartAdapter(DefaultPicoContainer.java:1015) ~[picocontainer-2.14.3.jar:na]
    at org.picocontainer.DefaultPicoContainer.startAdapters(DefaultPicoContainer.java:1008) ~[picocontainer-2.14.3.jar:na]
    at org.picocontainer.DefaultPicoContainer.start(DefaultPicoContainer.java:766) ~[picocontainer-2.14.3.jar:na]
    at org.sonar.api.platform.ComponentContainer.startComponents(ComponentContainer.java:91) ~[sonar-plugin-api-5.1.jar:na]
    at org.sonar.server.platform.ServerComponents$1.doPrivileged(ServerComponents.java:851) ~[sonar-server-5.1.jar:na]
    at org.sonar.server.user.DoPrivileged.execute(DoPrivileged.java:43) ~[sonar-server-5.1.jar:na]
    at org.sonar.server.platform.ServerComponents.executeStartupTasks(ServerComponents.java:847) ~[sonar-server-5.1.jar:na]
    at org.sonar.server.platform.Platform.executeStartupTasks(Platform.java:126) ~[sonar-server-5.1.jar:na]
    at org.sonar.server.platform.Platform.startLevel34Containers(Platform.java:122) ~[sonar-server-5.1.jar:na]
    at org.sonar.server.platform.Platform.doStart(Platform.java:81) ~[sonar-server-5.1.jar:na]
    at org.sonar.server.platform.PlatformServletContextListener.contextInitialized(PlatformServletContextListener.java:44) ~[sonar-server-5.1.jar:na]
    at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4720) [tomcat-embed-core-8.0.18.jar:8.0.18]
    at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5154) [tomcat-embed-core-8.0.18.jar:8.0.18]
    at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150) [tomcat-embed-core-8.0.18.jar:8.0.18]
    at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1409) [tomcat-embed-core-8.0.18.jar:8.0.18]
    at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1399) [tomcat-embed-core-8.0.18.jar:8.0.18]
    at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_45-internal]
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_45-internal]
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_45-internal]
    at java.lang.Thread.run(Thread.java:745) [na:1.8.0_45-internal]
2015.10.12 20:45:45 INFO  web[jruby.rack] jruby 1.7.9 (ruby-1.8.7p370) 2013-12-06 87b108a on OpenJDK 64-Bit Server VM 1.8.0_45-internal-b14 [linux-amd64]
2015.10.12 20:45:45 INFO  web[jruby.rack] using a shared (threadsafe!) runtime
2015.10.12 20:45:55 ERROR web[jruby.rack] initialization failed
org.jruby.rack.RackInitializationException: java.lang.NullPointerException
    at org.jruby.rack.RackInitializationException.wrap(RackInitializationException.java:31) ~[jruby-rack-1.1.13.2.jar:na]
    at org.jruby.rack.RackApplicationFactoryDecorator.init(RackApplicationFactoryDecorator.java:98) ~[jruby-rack-1.1.13.2.jar:na]
    at org.jruby.rack.RackServletContextListener.contextInitialized(RackServletContextListener.java:50) ~[jruby-rack-1.1.13.2.jar:na]
    at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4720) [tomcat-embed-core-8.0.18.jar:8.0.18]
    at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5154) [tomcat-embed-core-8.0.18.jar:8.0.18]
    at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150) [tomcat-embed-core-8.0.18.jar:8.0.18]
    at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1409) [tomcat-embed-core-8.0.18.jar:8.0.18]
    at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1399) [tomcat-embed-core-8.0.18.jar:8.0.18]
    at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_45-internal]
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_45-internal]
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_45-internal]
    at java.lang.Thread.run(Thread.java:745) [na:1.8.0_45-internal]
Caused by: java.lang.NullPointerException: null
    at org.sonar.server.ui.JRubyFacade.get(JRubyFacade.java:77) ~[sonar-server-5.1.jar:na]
    at org.sonar.server.ui.JRubyFacade.getDatabase(JRubyFacade.java:209) ~[sonar-server-5.1.jar:na]
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_45-internal]
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_45-internal]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_45-internal]
    at java.lang.reflect.Method.invoke(Method.java:497) ~[na:1.8.0_45-internal]
    at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(JavaMethod.java:440) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.javasupport.JavaMethod.invokeDirect(JavaMethod.java:304) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.java.invokers.InstanceMethodInvoker.call(InstanceMethodInvoker.java:52) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:306) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:136) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallNoArgNode.interpret(CallNoArgNode.java:60) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallNoArgNode.interpret(CallNoArgNode.java:60) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallNoArgNode.interpret(CallNoArgNode.java:60) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallNoArgNode.interpret(CallNoArgNode.java:60) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallOneArgNode.interpret(CallOneArgNode.java:57) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.BlockNode.interpret(BlockNode.java:71) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.RootNode.interpret(RootNode.java:129) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.INTERPRET_EVAL(ASTInterpreter.java:95) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.evalWithBinding(ASTInterpreter.java:184) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.RubyKernel.evalCommon(RubyKernel.java:1156) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.RubyKernel.eval(RubyKernel.java:1114) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.RubyKernel$INVOKER$s$0$3$eval.call(RubyKernel$INVOKER$s$0$3$eval.gen) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:180) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:296) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:72) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.FCallManyArgsNode.interpret(FCallManyArgsNode.java:60) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.IfNode.interpret(IfNode.java:118) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:139) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.DefaultMethod.call(DefaultMethod.java:182) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:306) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:136) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallNoArgNode.interpret(CallNoArgNode.java:60) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallOneArgNode.interpret(CallOneArgNode.java:57) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.BlockNode.interpret(BlockNode.java:71) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:139) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.DefaultMethod.call(DefaultMethod.java:182) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:306) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:136) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallNoArgNode.interpret(CallNoArgNode.java:60) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.AttrAssignOneArgNode.interpret(AttrAssignOneArgNode.java:33) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.BlockNode.interpret(BlockNode.java:71) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.IfNode.interpret(IfNode.java:116) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:139) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.DefaultMethod.call(DefaultMethod.java:182) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:306) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:136) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.VCallNode.interpret(VCallNode.java:88) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.BlockNode.interpret(BlockNode.java:71) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:161) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.DefaultMethod.call(DefaultMethod.java:190) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.RubyClass.finvoke(RubyClass.java:527) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.RubyBasicObject.send(RubyBasicObject.java:2590) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.RubyKernel.send(RubyKernel.java:2223) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.RubyKernel$INVOKER$s$send.call(RubyKernel$INVOKER$s$send.gen) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.JavaMethod$JavaMethodOneOrNBlock.call(JavaMethod.java:350) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:326) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:170) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallOneArgNode.interpret(CallOneArgNode.java:57) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.BlockNode.interpret(BlockNode.java:71) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:161) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.DefaultMethod.call(DefaultMethod.java:190) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:316) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.callBlock(CachingCallSite.java:145) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.callIter(CachingCallSite.java:154) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallNoArgBlockNode.interpret(CallNoArgBlockNode.java:64) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.BlockNode.interpret(BlockNode.java:71) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.RootNode.interpret(RootNode.java:129) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.INTERPRET_ROOT(ASTInterpreter.java:121) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.Ruby.runInterpreter(Ruby.java:838) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.Ruby.loadFile(Ruby.java:2727) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.load.ExternalScript.load(ExternalScript.java:66) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.load.LoadService.load(LoadService.java:359) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.RubyKernel.loadCommon(RubyKernel.java:1107) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.RubyKernel.load(RubyKernel.java:1094) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.RubyKernel$INVOKER$s$0$1$load.call(RubyKernel$INVOKER$s$0$1$load.gen) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:210) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:206) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:326) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:170) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.FCallOneArgNode.interpret(FCallOneArgNode.java:36) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.BlockNode.interpret(BlockNode.java:71) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:139) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.DefaultMethod.call(DefaultMethod.java:182) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:306) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:136) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallNoArgNode.interpret(CallNoArgNode.java:60) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:139) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.internal.runtime.methods.DefaultMethod.call(DefaultMethod.java:182) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:306) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:136) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.CallNoArgNode.interpret(CallNoArgNode.java:60) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.ast.RootNode.interpret(RootNode.java:129) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.evaluator.ASTInterpreter.INTERPRET_ROOT(ASTInterpreter.java:121) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.Ruby.evalScriptlet(Ruby.java:419) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.Ruby.evalScriptlet(Ruby.java:396) ~[jruby-complete-1.7.9.jar:na]
    at org.jruby.rack.rails.RailsRackApplicationFactory.createApplicationObject(RailsRackApplicationFactory.java:22) ~[jruby-rack-1.1.13.2.jar:na]
    at org.jruby.rack.DefaultRackApplicationFactory$1.create(DefaultRackApplicationFactory.java:98) ~[jruby-rack-1.1.13.2.jar:na]
    at org.jruby.rack.DefaultRackApplicationFactory$RackApplicationImpl.init(DefaultRackApplicationFactory.java:420) ~[jruby-rack-1.1.13.2.jar:na]
    at org.jruby.rack.DefaultRackApplicationFactory.getApplication(DefaultRackApplicationFactory.java:111) ~[jruby-rack-1.1.13.2.jar:na]
    at org.jruby.rack.SharedRackApplicationFactory.doInit(SharedRackApplicationFactory.java:31) ~[jruby-rack-1.1.13.2.jar:na]
    at org.jruby.rack.RackApplicationFactoryDecorator.init(RackApplicationFactoryDecorator.java:94) ~[jruby-rack-1.1.13.2.jar:na]
    ... 10 common frames omitted
2015.10.12 20:45:55 ERROR web[o.a.c.c.StandardContext] Error listenerStart
2015.10.12 20:45:55 ERROR web[o.a.c.c.StandardContext] Context [] startup failed due to previous errors
2015.10.12 20:45:55 WARN  web[o.a.c.l.WebappClassLoaderBase] The web application [ROOT] appears to have started a thread named [JRubyJIT-1] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 sun.misc.Unsafe.park(Native Method)
 java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
 java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
 java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
 java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
 java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
 java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
 java.lang.Thread.run(Thread.java:745)
2015.10.12 20:45:55 WARN  web[o.a.c.l.WebappClassLoaderBase] The web application [ROOT] appears to have started a thread named [JRubyJIT-2] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 sun.misc.Unsafe.park(Native Method)
 java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
 java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
 java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
 java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
 java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
 java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
 java.lang.Thread.run(Thread.java:745)
2015.10.12 20:45:55 INFO  web[o.a.c.h.Http11NioProtocol] Starting ProtocolHandler ["http-nio-0.0.0.0-9000"]
2015.10.12 20:45:55 INFO  web[o.s.s.a.TomcatAccessLog] Web server is started
2015.10.12 20:45:55 INFO  web[o.s.s.a.EmbeddedTomcat] HTTP connector enabled on port 9000
2015.10.12 20:45:55 WARN  web[o.s.p.ProcessEntryPoint] Fail to start web
```

will this be fixed with the 1.3 relase?

chears

**Comment by n3ziniuka5** [comment]
same here, 08eda37 worked fine, though

**Comment by arthepsy** [comment]
Thanks for noticing this quickly. Should be fixed with 1.2.1. Please, verify.

**Comment by n3ziniuka5** [comment]
works fine now, thanks
